client/settings/payments-and-transactions-section/statement-preview/index.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import CurrencyFactory from '@woocommerce/currency';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const icons = {
```

Comment:
Maybe it's just a personal choice. I would prefer to keep the icons in a separate file to make this one look a bit decluttered and contain the main component only.

Reply:
Yeah, that makes sense. It bothered me that the raw svg data was in this file taking up so much room. I've moved them to a new file.